Formula/imagemagick.rb

```diff
@@ -132,6 +136,14 @@ def caveats
   end
 
   test do
-    system "#{bin}/identify", test_fixtures("test.png")
+    system bin/"identify", test_fixtures("test.png")
+    # Test recommended features and delegates.
+    features = shell_output("#{bin}/convert -version")
+    assert_match "freetype", features if build.with? "freetype"
```

Comment:
Sure, I'll remove the `build.with?`. (Otherwise there will be six lines that are going to be executed.)